tests/unit/conversion.test.ts

```diff
@@ -0,0 +1,17 @@
+import { convertAmountFromHumanReadableToOnChain, convertAmountFromOnChainToHumanReadable } from "../../src";
+
+describe("conversion", () => {
+  test("it converts amount from human readable format to on chain format", () => {
+    const value = 500;
+    const decimals = 8;
+    const result = convertAmountFromHumanReadableToOnChain(value, decimals);
+    expect(result).toBe(50000000000);
+  });
+
+  test("it converts amount from on chain format to human readable format ", () => {
+    const value = 50000000000;
+    const decimals = 8;
+    const result = convertAmountFromOnChainToHumanReadable(value, decimals);
+    expect(result).toBe(500);
+  });
+});
```

Comment:
Do you want to check with decimal values?

Ideally, it probably should use the string version of the decimal, otherwise it may lose precision with 8 decimal points.

Reply:
done